web/ts/create-index.ts

```diff
@@ -24,6 +25,6 @@ const run = async () => {
     }
   }
 
-  await fs.writeFile(path.join(DATADIR, 'ms-index.json'), JSON.stringify(ms.toJSON()))
+  await fs.writeFile(process.argv[3], JSON.stringify(ms.toJSON()))
 }
 run()
```

Comment:
if you're going to do this, you need to add a default yarn run that would do something usefull.